code/modules/mob/living/living.dm

```diff
@@ -941,7 +939,11 @@
 	..(pressure_difference, direction, pressure_resistance_prob_delta)
 
 /mob/living/can_resist()
-	return !((next_move > world.time) || incapacitated(IGNORE_RESTRAINTS|IGNORE_STASIS))
+	if(next_move > world.time)
+		return FALSE
+	if(incapacitated(IGNORE_RESTRAINTS|IGNORE_STASIS))
```

Comment:
```suggestion
	if(incapacitated(IGNORE_RESTRAINTS|IGNORE_STASIS|IGNORE_GRAB))
```
Shouldn't this fix it? It seems this is what was blocking the resists.

Reply:
I thought of doing this but I don't know what effect it would have on other stuff, such as extinguishing yourself/unbuckling yourself from chairs/ect.

Reply:
Though at this point you might just want to do this:
```suggestion
	if(HAS_TRAIT(src, TRAIT_INCAPACITATED))
```

Reply:
> I thought of doing this but I don't know what effect it would have on other stuff, such as extinguishing yourself/unbuckling yourself from chairs/ect.

Seems fine, go for it with the last suggestion.

Reply:
> Though at this point you might just want to do this:

That is the equivalent to doing `if(incapacitated())` since that proc checks for the trait at the start and returns true if it's present.

Reply:
> > Though at this point you might just want to do this:
> 
> That is the equivalent to doing `if(incapacitated())` since that proc checks for the trait at the start and returns true if it's present.

But the problems are the other checks.
`TRAIT_INCAPACITATED` will handle situations such as the mob not being conscious, which we want to preserve.
The issue here is that if there is no `IGNORE_GRAB` in the `can_resist()` proc then an aggro grab will impede you form resisting, which is the present bug.

`incapacitated()` is not a great proc, mind you, but it was worse in the past, redefined all over and full of little checks.
I changed it so most source of mobility changes became traits, to allow for higher granularity. And this is one case of this, where we just want to check for incapacitated trait, but we don't care if the mob is restrained, being aggressively grabbed or in stasis. Those are not relevant for this, and the name `incapacitated()` is deceptive here. Those conditions restrain and block some actions, but do not fully incapacitate someone such as unconsciousness or paralysis do.

Reply:
Yeah the difference between knockout, paralysis, incapacitated, or stunned is kinda vague.  Might be better to have one with optional bitflags that determine special behavior.